firefox-desktop/firefox-desktop.model.lkml

```diff
@@ -53,6 +54,31 @@ explore: funnel_analysis {
   }
   sql_always_where: funnel_analysis.submission_date > "2010-01-01" ;;
 }
+
+explore: firefox_desktop_usage_2021 {
+  label: "Firefox Desktop Usage"
+  group_label: "Firefox-desktop"
```

Comment:
Should we stick these under the "KPIs" label?

Reply:
So my idea here was to have the Firefox Desktop Usage be a "separate" view (even though the KPIs are based on it) that will power a set of dashboards allowing people to drill into D/W/MAU. This would be separate to (but maybe linked from) the KPI dashboard that is focused only on displaying the company KPIs. Does that make sense?

Reply:
It does. Let's merge this as-is and I can take it on my end and make these explores available.